apps/easypid/src/config/copy.ts

```diff
@@ -20,3 +20,10 @@ export const copy = {
 export function useAppCopy() {
   return copy[CURRENT_APP_TYPE]
 }
+
+export function useAppIcon() {
```

Comment:
This is ok for now. But this is probably also something we can get from the app config at some pont

Reply:
How would you do this? Expo constants only returns the path from the root, like this:

`./assets/funke/icon.png`

But when I use it I need to use require, but that doesn't accept dynamic values. I also need relative paths.

How i use it now: 
```
            <Image height={96} width={96} src={require('../../../../assets/funke/icon.png')} />
```

this is not possible:
```
            <Image height={96} width={96} src={require(ExpoConstants.expoConfig?.icon)} />
```

Reply:
Ah okay 👍